src/definitions/elements/flag.less

```diff
@@ -39,4 +39,49 @@ i.flag:not(.icon)::before {
   }
 }
 
+each(@size-map, {
+  i.flag.@{key} {
+    font-size: 1.5em * @value;
+    vertical-align: middle;
+  }
+});
+
+/*rtl:begin:ignore*/
+
+each(@flags, {
```

Comment:
Somethings generates 2 same selectors:

```
i.flag.rainbow::before,
i.flag.rainbow::before,
i.flag.gay.pride::before {
```
```
i.flag.transgender::before

i.flag.rainbow::before,
i.flag.rainbow::before,
i.flag.gay.pride::before

i.flag.pirate::before
```

selectors should be not generated at all, they are not country flags

Reply:
i'll fix that. However they are designed as flags, even if not a country

Reply:
FYI I added `i.flag.crosseds`

Reply:
> FYI I added `i.flag.crosseds`

I see. Will remove crosseds, this is indeed not even a "flag" as meant to be used as a fui flag

Reply:
Fixed by #2466